ledger-core/application/testutils/launchnet/cloud.go

```diff
@@ -78,8 +82,44 @@ func (cr *CloudRunner) PrepareConfig() {
 	}
 }
 
+func prepareCloudForOneShotMode(confProvider *server.CloudConfigurationProvider) server.Server {
+	s, pulseDistributor := server.NewMultiServerWithoutPulsar(confProvider)
+	go func() {
+		// wait for starting all components
+		for !s.(*insapp.Server).Started() {
+			time.Sleep(time.Millisecond)
+		}
+
+		allNodes := make(map[reference.Global]struct{})
+		for _, conf := range confProvider.GetAppConfigs() {
+			cert, err := confProvider.CertificateFactory(nil, nil, conf.CertificatePath)
+			if err != nil {
+				panic(err)
+			}
+			allNodes[cert.GetCertificate().GetNodeRef()] = struct{}{}
+		}
+
+		pulseGenerator := testutils.NewPulseGenerator(uint16(confProvider.PulsarConfig.Pulsar.NumberDelta), nil, nil)
+		for i := 0; i < 2; i++ {
+			_ = pulseGenerator.Generate()
+			pulseDistributor.PartialDistribute(context.Background(), pulseGenerator.GetLastPulsePacket(), allNodes)
+		}
+	}()
+
+	return s
+}
+
 func (cr CloudRunner) SetupCloud() (func(), error) {
-	s := server.NewMultiServer(cr.ConfProvider)
+	return cr.SetupCloudCustom(false)
+}
+
+func (cr CloudRunner) SetupCloudCustom(pulsarOneShot bool) (func(), error) {
```

Comment:
```suggestion
type PulsarMode int

var (
   RegularPulsar PulsarMode = iota
   ManualPulsar
)

func (cr CloudRunner) SetupCloudCustom(mode PulsarMode) (func(), error) {
```

I'd rather have this function like this, instead of bool value

Reply:
What are other options? We need only two options so far.

Reply:
When you'll read code 
```
runner.SetupCloudCustom(RegularPulsar)
```
or 
```
runner.SetupCloudCustom(ManualPulsar)
```

it is easier to understand what is going on, rather then
```
runner.SetupCloudCustom(false)
```

If you want, PulsarMode could be bool, so using named constants will improve code readability

Reply:
But that works only for highest level. On lover levels it will look like runner.SetupCloudCustom(PulsarMode) - it doesn't give you more information.
But ok. I can do it